staging/src/k8s.io/dynamic-resource-allocation/deviceclass/extendedresourcecache/extendedresourcecache.go

```diff
@@ -102,6 +102,7 @@ func (c *ExtendedResourceCache) OnAdd(obj interface{}, isInInitialList bool) {
 		utilruntime.HandleErrorWithLogger(c.logger, nil, "Expected DeviceClass", "actual", fmt.Sprintf("%T", obj))
 		return
 	}
+	c.logger.V(5).Info("DeviceClass added", "deviceClass", klog.KObj(deviceClass))
```

Comment:
I found these useful during debugging. I can also take it out.

Reply:
i think it is fine as it is V(5), also there won't be many, frequent device class changes in a cluster.